devops/playbooks/aws-ci-startup.yml

```diff
@@ -1,10 +1,39 @@
 #!/usr/bin/env ansible-playbook
 
-- name: Create CI remote hosts for testing SecureDrop config.
+- name: Provision build docker image
   hosts: localhost
   become: no
   gather_facts: true
   tasks:
+    - name: Pull in systemd enabled docker image
+      docker_image:
+        name: "{{ docker_image_sd_build }}"
+      tags: docker
+      when: "{{ docker_containers != [] }}"
+
+    - name: Start build container
+      docker_container:
+        name: "{{ item }}"
+        hostname: "{{ item }}"
+        image: "{{ docker_image_sd_build }}"
+        command: /lib/systemd/systemd
+        devices:
+          - "/sys/fs/cgroup:/sys/fs/cgroup:ro"
+        privileged: true
```

Comment:
`you shouldn't need this ` 

Yeah .. i "shouldnt" but i kept trying to make it work without it and it was failing. That honestly could have been in combination with another setting I was using at the time. I'm not dying to go back and debug though in this ticket though but I'll be happy to debug in another PR/ticket. Since the image is primarily running in CI in an environment outside of our server's I dont see this as urgent to get into this PR.